nerdlets/e2m-gui-nerdlet/form-components/rule-description-validator.js

```diff
@@ -1,90 +1,88 @@
-import React from "react";
-import PropTypes from "prop-types";
-import { TextField, Spinner } from "nr1";
+import React from 'react';
+import PropTypes from 'prop-types';
+import { TextField } from 'nr1';
 
 class RuleDescriptionValidator extends React.Component {
-	constructor(props) {
-		super(props);
-		this.state = {
-			ruleDescription: null,
-			validationTimeout: null,
-			validating: false,
-			validated: null
-		};
-	}
+  constructor(props) {
+    super(props);
+    this.state = {
+      ruleDescription: null
+    };
+  }
 
-	clear() {
-		this.setState({
-			ruleDescription: null,
-			validationTimeout: null,
-			validating: false,
-			validated: null
-		});
-	}
+  clear() {
+    this.setState({
+      ruleDescription: null
+    });
+  }
 
-	setDescription(userInputtedRuleDescription) {
-		/* Remove dangerous characters from name */
-		const ruleDescription = userInputtedRuleDescription.replace(
-			/[^a-zA-Z0-9\.]/g,
-			""
-		);
+  setDescription(userInputtedRuleDescription) {
+    /* Remove dangerous characters from name */
+    /* eslint-disable no-useless-escape */
+    const ruleDescription = userInputtedRuleDescription.replace(
+      /[^a-zA-Z0-9\.]/g,
+      ''
+    );
+    /* eslint-enable */
 
-		if (this.validRuleDescription(ruleDescription)) {
-			this.props.setValidatedDescription(ruleDescription);
-			this.setState({ ruleDescription, valid: true });
-		} else {
-			this.setState({ ruleDescription, valid: false });
-		}
-	}
+    if (this.validRuleDescription(ruleDescription)) {
+      this.props.setValidatedDescription(ruleDescription);
+      this.setState({ ruleDescription });
+    } else {
+      this.setState({ ruleDescription });
+    }
+  }
 
-	validRuleDescription(name) {
-		return true;
-	}
+  /* eslint-disable no-unused-vars */
+  validRuleDescription(name) {
+    return true;
+  }
+  /* eslint-enable */
```

Comment:
Signature

Reply:
I'm assuming that there's a future for that function and the related input. That said, the linting rules would disallow the name input that's unused.